src/components/molecules/Building.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+	<div>
+		<BuildingImg1/>
```

Comment:
No need for this component, you will not use it anywhere else. It will be used in Building molecule only, so simplify 2 images coponent to be not a 3 files, but 1 file wtih styling